packages/components/src/internal/components/permissions/GroupDetailsPanel.tsx

```diff
@@ -13,58 +13,52 @@ import { UserProperties } from '../user/UserProperties';
 
 import { useServerContext } from '../base/ServerContext';
 
+import { useAppContext } from '../../AppContext';
+
 import { EffectiveRolesList } from './EffectiveRolesList';
 
 import { Principal, SecurityPolicy, SecurityRole } from './models';
 import { MembersList } from './MembersList';
 
 interface Props {
     displayCounts?: boolean;
-    getAuditLogData: (columns: string, filterCol: string, filterVal: string | number) => Promise<string>;
     isSiteGroup: boolean;
     members?: Member[];
     policy: SecurityPolicy;
-    principal: Principal;
+    principal: Principal; // TODO: Is "principal" required or not? The code seems confused about this
     rolesByUniqueName: Map<string, SecurityRole>;
     showPermissionListLinks?: boolean;
 }
 
 export const GroupDetailsPanel: FC<Props> = memo(props => {
-    const {
-        getAuditLogData,
-        principal,
-        members,
-        isSiteGroup,
-        displayCounts = true,
-        showPermissionListLinks = true,
-    } = props;
+    const { principal, members, isSiteGroup, displayCounts = true, showPermissionListLinks = true } = props;
     const [created, setCreated] = useState<string>('');
+    const { api } = useAppContext();
     const { user } = useServerContext();
 
     const loadWhenCreated = useCallback(async () => {
         try {
-            const createdState = await getAuditLogData('Date,group/UserId', 'group/UserId', principal.userId);
-
+            const createdState = await api.security.getAuditLogData('group/UserId', principal.userId);
+            // TODO: Surely need a comment about -7
```

Comment:
Will investigate. @RosalineP you may know.

Reply:
This truncates the timestamp to date and time, eg "2023-12-20 12:57:32.748" to "2023-12-20 12:57"

Reply:
Great thanks. I've updated the API endpoint wrapper to return the `formattedValue` if available and removed this slicing logic.